app/config.py

```
@@ -2,142 +2,142 @@
-from pydantic_settings import BaseSettings
 from enum import Enum
-from typing import List, Optional, Dict, Any
+from typing import Any, Dict, List, Literal, Optional
+
+from pydantic_settings import BaseSettings
 
 
 class GitHosting(str, Enum):
-    GITLAB = "gitlab"
-    GITHUB = "github"
+	GITLAB = "gitlab"
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)